lib/geolocation/simulator.js

```diff
@@ -14,27 +14,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+'use strict';
+
 var console = require('console')
 
 var Simulator = function () {
-   throw("TODO: unimplemented");
-}
+  this.value = {
+    latitude: 0,
+    longitude: 0
+  };
+};
 
 Simulator.prototype.read = function (callback) {
-  throw("TODO: unimplemented");
-  if (callback) return callback(null, this.value)
+  this.value.latitude = Math.random() * 180 - 90;
+  this.value.longitude = Math.random() * 360 - 180;
+  if (callback) return callback(null, this.value);
```

Comment:
please revert this, do do cleanup change before of after and then I think we're fine

you can use make lint on master to do that
and then rebase and I'll merge the 3 patches 

is it ok ?

Reply:
I'm sorry, I don't quite understand what you want me to revert... Should the callback not be called?

I've also fixed most linting errors, but it also complains about the `console.log` in `index.js`, even though it doesn't for the other sensors - do you know why?

Reply:
I meant about the missing trailing semi col , I borrowed from color-sensor-js ..

Reply:
But... There _is_ a trailing semi colon there?